app/models/mixins/purging_mixin.rb

```diff
@@ -206,6 +206,8 @@ def purge_in_batches(conditions, window, total = 0, total_limit = nil)
           break if batch_ids.empty?
 
           current_window = batch_ids.size
+          _log.info("Purging Associated Records #{current_window} #{table_name.humanize}.")
```

Comment:
Minor, but this reads strangely to me.  How do we log other parts of the purging...I'm also concerned this will be kind of verbose since it's within the loop.

Reply:
yeah, was debating it too.

I'll move the Purging log line before this conditional so it includes both the associated records for the batch and the parent batch, like it did previously.